src/docs/san-tokens/how-to-buy-san-tokens/index.md

```diff
@@ -1,32 +1,43 @@
+## Step 1: Install and Set Up MetaMask
+
+For instructions on how to install and set up MetaMask, visit [MetaMask.io](https://metamask.io). You will need an Ethereum address to hold SAN tokens and interact with the Santiment platform. You can create a new wallet in MetaMask, import an existing wallet, or connect MetaMask to a hardware wallet like **Ledger** or **Trezor**.
 
 <mark>__Result of Step 1:__ Installed and working Metamask</mark>
 
-## Step 2: Add funds to MetaMask wallet
+## Step 2: Add Funds to MetaMask Wallet
 
-To get SAN tokens, you will need to swap them for ETH or another ERC20 token. You may acquire ETH from a centralized exchange like [Bitfinex](https://bitfinex.com/t/ETH:USD) and then transfer it to your MetaMask wallet.
+To acquire SAN tokens, you will need to swap them for ETH or another ERC20 token. You can obtain ETH from a centralized exchange like [Bitfinex](https://bitfinex.com/t/ETH:USD) and then transfer it to your MetaMask wallet.
 
 <mark>__Result of Step 2:__ Metamask wallet with ETH or ERC-20 token balance</mark>
 
 ## Step 3: Swap ETH to SAN Tokens
 
-You can use decentralized exchanges or decentralized exchange aggregators to swap ETH or some other ERC20 token for SAN tokens. They all work very similarly. Connect your Metamask wallet first, and then swap ETH for SAN tokens.
+You can use decentralized exchanges or decentralized exchange aggregators to swap ETH or another ERC20 token for SAN tokens. These platforms operate in a similar manner. First, connect your Metamask wallet, and then proceed to swap ETH for SAN tokens.
+
+### Decentralized Exchanges
+
+Decentralized exchanges (DEX) are platforms that allow users to trade cryptocurrencies without relying on a centralized authority to manage their assets. Instead, transactions are executed directly between users through smart contracts on a blockchain network. This provides increased security, privacy, and control over one's funds.
+
+Some popular decentralized exchanges include Uniswap, Kyber, and Bancor. These platforms typically offer features such as liquidity provision, yield farming, and token swapping.
+
+To get started with using a decentralized exchange, you will need a cryptocurrency wallet that supports the blockchain network on which the DEX operates. For example, if you want to use Uniswap, you will need a wallet that supports Ethereum, such as MetaMask or Trust Wallet.
+
+Once you have set up your wallet, you can connect it to the DEX platform and start trading. Keep in mind that fees and transaction times may vary depending on the network congestion and the specific DEX you are using.
 
-### Decentralized exchanges
+In summary, decentralized exchanges offer a more secure and private way to trade cryptocurrencies compared to centralized exchanges. However, they may require a bit more technical knowledge to navigate and use effectively.
```

Comment:
Those paragraphs were not in the previous version. I wonder do we need so much text to describe a DEX here? Does this provide value to the reader?